tsl/test/isolation/expected/dist_restore_point.out

```diff
@@ -0,0 +1,333 @@
+step s2_setup: SELECT created FROM create_distributed_hypertable('disttable', 'time', 'device');
+created        
+
+t              
+
+starting permutation: s3_lock_enable s1_create_dist_rp s2_insert s3_lock_count s3_lock_release
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_1    localhost      55432          cdrp_1         f              f              f              
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_2    localhost      55432          cdrp_2         f              f              f              
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_3    localhost      55432          cdrp_3         f              f              f              
+step s3_lock_enable: SELECT debug_waitpoint_enable('create_distributed_restore_point_lock');
+debug_waitpoint_enable
+
+               
+step s1_create_dist_rp: SELECT restore_point > pg_lsn('0/0') as valid_lsn FROM create_distributed_restore_point('s1_test'); <waiting ...>
+step s2_insert: INSERT INTO disttable VALUES ('2019-08-02 10:45', 0, 0.0); <waiting ...>
+step s3_lock_count: 
+	SELECT waitpoint_locks('create_distributed_restore_point_lock') as cdrp_locks, 
+	       remote_txn_locks() as remote_txn_locks;
+
+cdrp_locks     remote_txn_locks
+
+2              2              
+step s3_lock_release: SELECT debug_waitpoint_release('create_distributed_restore_point_lock');
+debug_waitpoint_release
+
+               
+step s1_create_dist_rp: <... completed>
+valid_lsn      
+
+t              
+t              
+t              
+t              
+step s2_insert: <... completed>
+
+starting permutation: s2_begin s2_update s3_lock_enable s1_create_dist_rp s2_commit s3_lock_count s3_lock_release
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_1    localhost      55432          cdrp_1         f              f              f              
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_2    localhost      55432          cdrp_2         f              f              f              
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_3    localhost      55432          cdrp_3         f              f              f              
+step s2_begin: BEGIN;
+step s2_update: UPDATE disttable SET temp = temp + 1.0;
+step s3_lock_enable: SELECT debug_waitpoint_enable('create_distributed_restore_point_lock');
+debug_waitpoint_enable
+
+               
+step s1_create_dist_rp: SELECT restore_point > pg_lsn('0/0') as valid_lsn FROM create_distributed_restore_point('s1_test'); <waiting ...>
+step s2_commit: COMMIT; <waiting ...>
+step s3_lock_count: 
```

Comment:
Don't we want to show the lock counts before COMMIT? I.e., we want to see that the restore point session blocks on the remote_txn lock instead of the waitpoint lock.

Reply:
This is what it does, because waitpoint lock is taken after the remote_txn lock, we can see them both. I've updated the test to see those counters before the commit